### PR TITLE
CSharp Connection String implementation

### PR DESCRIPTION
**Pull request by toddbellMSFT**

### CODE REVIEW

File: targets/csharp/source/PlayFabSDK/UnittestRunner/UnittestRunner.csproj [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>netcoreapp1.1;netcoreapp2.0;net45</TargetFrameworks>
```
Comment (amccalib) [inline targets/csharp/source/PlayFabSDK/UnittestRunner/UnittestRunner.csproj:5]: Are the netcoreapp additions intentional?

File: targets/csharp/source/PlayFabSDK/source/PlayFabSDK.csproj.ejs [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;netstandard1.1;net45;portable45-net45+win8+wpa81</TargetFrameworks>
```
Comment (amccalib) [inline targets/csharp/source/PlayFabSDK/source/PlayFabSDK.csproj.ejs:4]: Same as above. Are these changes intentional?

File: targets/csharp/source/PlayFabSDK/source/PlayFabSettings.cs.ejs [hunk 3]
```diff
@@ -37,6 +37,8 @@ namespace PlayFab
         public static string TitleId { get { return staticSettings.TitleId; } set { staticSettings.TitleId = value; } }
         #endregion Deprecated staticSettingsredirect properties
 
+        public static string ConnectionString { get { return staticSettings.ConnectionString; } set { staticSettings.ConnectionString = value; } }
```
Comment (amccalib) [inline targets/csharp/source/PlayFabSDK/source/PlayFabSettings.cs.ejs:40]: Given the Obsolete attributes above, should we be adding this?  It sounds like we developers should just be referencing staticSettings instead of this property directly.

**Review by amccalib — CHANGES_REQUESTED** [review]

File: targets/csharp/source/PlayFabSDK/UnittestRunner/UnittestRunner.csproj (hunk 1 above)
Comment (toddbellMSFT) [reply targets/csharp/source/PlayFabSDK/UnittestRunner/UnittestRunner.csproj:5]: Yup, netcoreapp 1.1 and 2.0 apparently have hit end of life :( 

https://dotnet.microsoft.com/download/dotnet/2.0/runtime?utm_source=getdotnetcore&utm_medium=referral 

I got a hang when running 1.1, and 2.0 spits our this url that mentions this is no longer supported by microsoft.

File: targets/csharp/source/PlayFabSDK/source/PlayFabSDK.csproj.ejs (hunk 2 above)
Comment (toddbellMSFT) [reply targets/csharp/source/PlayFabSDK/source/PlayFabSDK.csproj.ejs:4]: portable45 had an issue, and so did the win8 stuff. I put back netstandard 2.0 and 1.1 in this project file and it still ran fine without issues so I can add those back (netcore seems to be what is end of life, standard seems to still be around)

**Review by amccalib — APPROVED** [review]
